vi/cs-230-convolutional-neural-networks.md

```diff
@@ -0,0 +1,716 @@
+**Convolutional Neural Networks translation** [[webpage]](https://stanford.edu/~shervine/teaching/cs-230/cheatsheet-convolutional-neural-networks)
+
+<br>
+
+**1. Convolutional Neural Networks cheatsheet**
+
+&#10230;Convolutional Neural Networks cheatsheet
+
+<br>
+
+
+**2. CS 230 - Deep Learning**
+
+&#10230; CS 230 - Deep Learning
+
+<br>
+
+
+**3. [Overview, Architecture structure]**
+
+&#10230; [Tổng quan, Kết cấu kiến trúc]
+
+<br>
+
+
+**4. [Types of layer, Convolution, Pooling, Fully connected]**
+
+&#10230; [Các kiểu tầng (layer), Tích chập, Pooling, Kết nối đầy đủ]
```

Comment:
```suggestion
&#10230; [Các kiểu tầng (layer), Tích chập, Gộp, Kết nối đầy đủ]
```

Reply:
I don't think that is a proper translation for "pooling". Do you have any references?

Reply:
Can't excerpt the detail but it is quite popular in my university (HUST)

Reply:
Thank you. I think it's better to keep it as "pooling"